tutorials/sheets_sleep.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Sheets CDC 500 Sleep Sorting
+nav_order: 2
+parent: Tutorials
+---
+
+# Sorting the CDC 500 cohort on sleep 
+
+## Introduction
+
+The Data Commons API enables easy access to health data for the 500 cities the CDC has prioritized for public health information tracking. This tutorial will walk you through accessing and analyzing that information and scoring each of the cities according to adults residents’ excellence in sleep habits.
+
+## Step 1: Setup
+Pull up Google Sheets and create a new, blank spreadsheet. You can title it `Data Commons CDC 500 sleep analysis` or any other name of your choosing.
+
+![](/assets/images/tutorials/sheets_sleep_tutorial_1.png)
+
+To enable the Data Commons API in your spreadsheet, ensure that the Data Commons extension is installed and available under the Extensions tab in the main menu. Hover over the Data Commons menu item, then click on the Fill Place DCIDs option.
+
+![](/assets/images/tutorials/sheets_sleep_tutorial_2.png)
+
+## Step 2: Get the DCIDs and names of cohort members
+Data Commons provides the method `DCCOHORTMEMBERS` for obtaining the members of a Data Commons cohort. Here, you’ll use this method to retrieve the cities in the CDC 500 cohort. Start by double-clicking on A1 in the chart and enter `CDC500_City`.
+
+![](/assets/images/tutorials/sheets_sleep_tutorial_3.png)
+
+In cell B1, enter `=DCCOHORTMEMBERS(A1)`. The output should look like this:
+
+![](/assets/images/tutorials/sheets_sleep_tutorial_4.png)
+
+To get the names of these cities, enter `=DCGETNAME(B1:B500)` into cell C1. The output should look like this:
+
+![](/assets/images/tutorials/sheets_sleep_tutorial_5.png)
+
+## Step 3: Obtain the sleep health level for each city.
+We will use the percentage of chronically restless residents in each city to determine using the `DCGET` method with the statistical variable `Percent_Person_SleepLessThan7Hours`. (More information on statistical variables is available in the glossary. Note that this is one of quite a number of statistical variables that can be measured for the CDC 500.) Enter `=DCGET(B1:B500, "Percent_Person_SleepLessThan7Hours", "2016")` into cell D1 in your spreadsheet. The output should look like this:
```

Comment:
hm, perhaps "to determine" in the first sentence can be removed?

chronically restless residents.. funny :)

Reply:
add a link to the glossary? i also wonder if there's a way of pointing the reader to the other statistical variables mentioned without being overwhelming.

Reply:
Glossary linked. After some thought, I decided to just remove the section about other statistical variables available--I'm not sure that it really adds anything to this tutorial.